src/customize-pop/keyboard-settings.md

```diff
@@ -0,0 +1,70 @@
+# Keyboard Settings
+
+Option to set input Sources, input source switching, special character entry options, and keyboard shortcuts are located in `Settings` ➞ `Keyboard`.
+
+![Keyboard Settings](/images/keyboard-settings/keyboard-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Input Sources | Choose multiple input languages and layouts for a single keyboard. Sources can be moved up or down to determine priority. |
+| Input Source Switching | Set a keyboard shortcut for switching between your chosen sources. |
+| Special Character Key | Designate keys that enable entry of special characters. |
+| Keyboard Shortcuts | View, edit, and add keyboard shortcuts. |
+
+## Adding an Input Source
+
+1. Click the `+` to display available input sources. Select an input source from the list, or click the three vertical dots to perform a keyword search.
+
+    ![Display Sources](/images/keyboard-settings/display-sources.png)
+
+2. Select a language from the list.
+
+    ![Select Source](/images/keyboard-settings/select-source.png)
+
+3. Select a keyboard layout and click `Add`.
+
+> **Keyboard Layout vs. Keyboard Input Language**: A keyboard language refers to the available keys required to input the required characters and symbols for a given language. A keyboard layout is the physical organization of those keys.
+
+![Select Layout](/images/keyboard-settings/select-layout.png)
+
+Display keyboard layouts by clicking the three vertical dots on the right and clicking `View Keyboard Layout`
+
+![View Keyboard Layout](/images/keyboard-settings/view-keyboard-layout.png)
+
+Type while viewing the keyboard layout to verify accurate key registration.
```

Comment:
The word `registration` implies recording something. I would suggest changing to a more passive word such as `recognition`, or a phrase like `keystroke detection`, depending on what use case you're trying to refer to.